integrations-examples/pipecat-bot/main.py

```
@@ -0,0 +1,103 @@
+from pipecat.processors.frameworks.rtvi import (
+    RTVIConfig,
+    RTVIObserver,
+    RTVIProcessor,
+    RTVIObserverParams,
+)
+from pipecat.runner.types import RunnerArguments
+from pipecat.runner.utils import create_transport
+from pipecat.services.gladia.config import GladiaInputParams, MessagesConfig
+from pipecat.services.gladia.stt import GladiaSTTService
+from pipecat.transports.base_transport import BaseTransport, TransportParams
+from pipecat.transports.network.small_webrtc import SmallWebRTCTransport
```

Ruff (0.12.2)

`pipecat.transports.network.small_webrtc.SmallWebRTCTransport` imported but unused

Remove unused import: `pipecat.transports.network.small_webrtc.SmallWebRTCTransport`

(F401)